datafusion/expr/src/expr_fn.rs

```diff
@@ -215,6 +215,18 @@ pub fn count(expr: Expr) -> Expr {
     ))
 }
 
+/// Create an expression to represent the count_null() aggregate function
```

Comment:
I think this function is creating a function like `COUNT(IS NULL x)` rather than a `count_null` function